components/Navigation/NavigationContainer/__tests__/index.test.tsx

```diff
@@ -0,0 +1,39 @@
+import userEvent from '@testing-library/user-event';
+import { render, screen } from '@testing-library/react';
+import { IntlProvider } from 'react-intl';
+import NavigationContainer from '../';
+import type { FC } from 'react';
+
+const NavigationContent: FC = () => (
+  <ul>
+    <li>Navigation item 1</li>
+    <li>Navigation item 2</li>
+    <li>Navigation item 3</li>
+  </ul>
+);
+const label: string = 'Navigation';
+let isOpen: boolean = false;
```

Comment:
```suggestion
```

I think constants can be removed from here and passed as a props to `NavigationContainer` since its not used more than one place

Reply:
We actually use it ?

Reply:
it's being used in the below test